services/app-api/src/emailer/emails/sendQuestionCMSEmail.ts

```diff
@@ -0,0 +1,78 @@
+import { packageName as generatePackageName } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+import { formatCalendarDate } from '../../../../app-web/src/common-code/dateHelpers'
+import { pruneDuplicateEmails } from '../formatters'
+import type { EmailConfiguration, EmailData, StateAnalystsEmails } from '..'
+import type { ProgramType, CMSUserType } from '../../domain-models'
+import {
+    stripHTMLFromTemplate,
+    renderTemplate,
+    findContractPrograms,
+} from '../templateHelpers'
+import { submissionQuestionResponseURL } from '../generateURLs'
+import type { ContractRevisionWithRatesType } from '../../domain-models/contractAndRates'
+
+export const sendQuestionCMSEmail = async (
+    contractRev: ContractRevisionWithRatesType,
+    stateAnalystsEmails: StateAnalystsEmails,
+    cmsRequestor: CMSUserType,
+    config: EmailConfiguration,
+    statePrograms: ProgramType[],
+    dateAsked: Date,
+    roundNumber: number
+): Promise<EmailData | Error> => {
+    let receiverEmails = [...stateAnalystsEmails, ...config.devReviewTeamEmails]
+    if (cmsRequestor.divisionAssignment === 'DMCP') {
+        receiverEmails.push(...config.dmcpEmails)
+    } else if (cmsRequestor.divisionAssignment === 'OACT') {
+        receiverEmails.push(...config.oactEmails)
```

Comment:
I think we are missing `DMCO` primary inbox `config.dmcoEmails` here.

AC 1
```
If a CMS user from ANY division (DMCO, DMCP, or OACT) submits a round of questions for a submission 
Then the CMS DMCO users assigned to the state receives an automated confirmation email.
```

Reply:
Would `config.dmcoEmails` go to all DMCO users? My understanding of that AC was to only send to the state analysts for the particular state the question is about